lib/swig/tick/hawkes/simulation/hawkes_kernels.i

```diff
@@ -14,6 +14,13 @@ class HawkesKernel {
   double get_primitive_value(double t);
   double get_primitive_value(double s, double t);
   virtual double get_norm(int nsteps = 10000);
+  virtual double get_convolution(
+       const double time, 
+       const ArrayDouble &timestamps, 
+       double *const bound);
+  virtual double get_primitive_convolution(
+       const double time, 
+       const ArrayDouble &timestamps);
```

Comment:
This should be needed only if you want to access these functions form Python. Otherwise, it should not be.

Reply:
Hi @Mbompr 

I believe @PhilipDeegan has already resolved the issue. We can close this PR.